hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/AssociatePrecompileSuite.java

```diff
@@ -443,7 +443,7 @@ private HapiSpec invalidSingleAbiCallConsumesAllProvidedGas() {
                             .getTransactionRecord(INVALID_SINGLE_ABI_CALL_TXN)
                             .getContractCallResult()
                             .getGasUsed();
-                    assertEquals(99011, gasUsed);
+                    assertEquals(99205, gasUsed);
```

Comment:
Newer versions of solidity compile the bytecode slightly differently.  If we have code that over-fits to specific gas values then we will always will need to change it.

If we used the optimize flags it will change even mode.